lib/util/add_task_dialogue_box.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import 'package:todo_app/util/dialogue_box_button.dart';
+
+class DialogueBox extends StatelessWidget {
+  final controller;
```

Comment:
I believe your linter will yell at you for not having explicit type.
This should be
```dart
final TextEditingController controller;
```
instead